frontend/src/js/index.jsx

```diff
@@ -24,3 +25,7 @@ export default store;
 // Particle Js Decorations
 window.particlesJS('particle', particlesOption);
```

Comment:
We should probably get rid of this, it's not so professional. Thoughts @bunsenmcdubbs ?

Reply:
It's for decoration purpose. The configuration can be customized here: http://vincentgarreau.com/particles.js/. If it looks bad then I should probably remove it.

Reply:
I agree with Michael - we should remove it. Seems irrelevant